runtime/alloc/alloc.c

```diff
@@ -13,6 +13,7 @@
 REGISTER_ARENA(youngspace, 0);
 REGISTER_ARENA(oldspace, 1);
 REGISTER_ARENA(nogcspace, 2);
+REGISTER_ARENA(alwaysgcspace, 3);
```

Comment:
we could save some memory usage if we don't make this space have two parts.

Reply:
I agree and changed this